components/Ink/src/MDCSimpleInkView.m

```diff
@@ -0,0 +1,95 @@
+/*
+ Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MDCSimpleInkView.h"
+#import "MDCSimpleInkGestureRecognizer.h"
+#import "private/MDCSimpleInkLayer.h"
+
+@interface MDCSimpleInkView ()
+
+@property(nonatomic, strong) MDCSimpleInkLayer *inkLayer;
+
+@end
+
+@implementation MDCSimpleInkView
+
+- (instancetype)initWithFrame:(CGRect)frame {
+  self = [super initWithFrame:frame];
+  if (self) {
+    _inkColor = [UIColor colorWithWhite:0 alpha:0.08f];
+  }
+  return self;
+}
```

Comment:
If you implement one designated initializer, you typically want to implement them all.
Add initWithCoder: that handles _inkColor.

Reply:
Done.